src/devices/Qmc5883l/samples/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Numerics;
+using System.Threading;
+using Iot.Device.Qmc5883l;
+
+I2cConnectionSettings settings = new(1, Qmc5883l.DefaultI2cAddress);
+using I2cDevice device = I2cDevice.Create(settings);
+using (Qmc5883l sensor = new(device))
+{
+    // Configure the sensor to match our needs.
+    // Make sure to set the device in Continuous mode if you plan on reading any data.
```

Comment:
I'd suggest to make it a default in the constructor and optionally (only if you find it useful) provide a flag to not touch the mode or make it standby by default